test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Configurations/OpenApiinfoConfigs.cs

```diff
@@ -0,0 +1,16 @@
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests.Configuretions
```

Comment:
I don't think this file is necessary either. Would you please remove this file as well?